bridge/bridge.go

```diff
@@ -0,0 +1,14 @@
+package bridge
+
+import (
+	"github.com/giantswarm/flanneltpr/docker"
+)
+
+// Bridge holds information used for the execution of the Docker entrypoint of
+// https://github.com/giantswarm/k8s-network-bridge.
+type Bridge struct {
+	// Config contains network bridge configuration.
+	Config Config `json:"config" yaml:"config"`
+	// Docker describes the docker image running k8s-network-bridge.
+	Docker docker.Docker `json:"docker" yaml:"docker"`
```

Comment:
Both `Bridge` and `Flannel` have this structure of `Config` and `Docker` fields. I found it nice to separate that. Config is configuration of the running component, everything outside is the way to run it - in this case with docker.